src/main/java/bitbot/ToDos.java

```diff
@@ -0,0 +1,31 @@
+package bitbot;
+
+import java.time.format.DateTimeFormatter;
+
+public class ToDos extends Task {
```

Comment:
Consider renaming "ToDos" to "Todo": abbreviations aren't capitalized so it makes more sense for lowercase "d" in my opinion, and I feel singular "Todo" is more accurate to the class's function than plural "Todos".

Reply:
I concur with sumomomomomo! I feel that "Todo" would be more suited.